example/main.tf

```diff
@@ -14,24 +14,8 @@
  * limitations under the License.
  */
 
-locals {
-  credentials_file_path = var.credentials_file_path
-}
-
-/******************************************
-  Provider configuration
- *****************************************/
-provider "google" {
```

Comment:
The provider should still be configured in the example, with an appropriate `version` value and no `credentials` argument.